src/Notifier.php

```diff
@@ -124,21 +124,32 @@ public function buildNotice($exc)
      */
     protected function postNotice($url, $data)
     {
-        $options = [
-            'http' => [
-                'header' => "Content-type: application/json\r\n",
-                'method' => 'POST',
-                'content' => $data,
-                'ignore_errors' => true,
-            ],
-        ];
-        $context = stream_context_create($options);
-        $respData = file_get_contents($url, false, $context);
-
-        return [
-            'headers' => $http_response_header,
-            'data' => $respData,
-        ];
+        try {
+            $respData = false;
+            $options = [
+                'http' => [
+                    'header' => "Content-type: application/json\r\n",
+                    'method' => 'POST',
+                    'content' => $data,
+                    'ignore_errors' => true,
+                ],
+            ];
+            $context = stream_context_create($options);
+            /* file_get_contents: https://secure.php.net/file_get_contents
+             *
+             * E_WARNING level error is generated if filename cannot be found,
+             * maxlength is less than zero,
+             * or if seeking to the specified offset in the stream fails. 
+            */
+            $respData = file_get_contents($url, false, $context);
+        } catch (Exception $e) {
+            throw $e;
```

Comment:
I threw it because I didn't know what you might want to do with it.  Different people treat exceptions differently.

Basically, I keep getting this message over and over again:
[Fri Oct 16 19:27:49.711275 2015] [:error] [pid 28852] [client 172.31.27.19:53725] PHP Warning:  file_get_contents(): SSL: Broken pipe in /var/app/current/vendor/airbrake/phpbrake/src/Notifier.php on line 137